### DISCUSSION TITLE
Trying to remove onPageChange warning.

### DISCUSSION THREAD

**Discussion by even996** [material-table-core/core#361] [Q&A]
Hi!
Am trying to remove this warning.  

vendors~main.chunk.js:281042 Warning: Failed prop type: The prop `onPageChange` is marked as required in `ForwardRef(TablePagination)`, but its value is `undefined`.
    at TablePagination (http://localhost:3000/static/js/vendors~main.chunk.js:91674:37)

Everything else works just fine. 
Am using @material-ui/core 4.11.4 and material table 1.69.3

I think its somewhere here i need to add onChangePage but am not sure how.

![image](https://user-images.githubusercontent.com/33283405/136372406-8d22631c-9b41-4c53-852e-d5ead84ed6d5.png)

`<MaterialTable
			icons={tableIcons}
			title="Lines status list"
			data={listData}
			columns={columns}
		/>`

Thanks in advance :)

**Comment by mukaofssn** [comment]
Not an answer but I am also getting the following warnings and the stack traces seem to occur from within MaterialTable > DragDropContext:

_Failed prop type: The prop `onChangePage` of `ForwardRef(TablePagination)` is deprecated. Use the `onPageChange` prop instead.
Failed prop type: The prop `onChangeRowsPerPage` of `ForwardRef(TablePagination)` is deprecated. Use the `onRowsPerPageChange` prop instead.
Failed prop type: The prop `onPageChange` is marked as required in `ForwardRef(TablePagination)`, but its value is `undefined`._

Am using the following:
```
"@material-table/core": "^0.2.10",
"@mui/material": "^5.2.6",
```
